src/snowflake/snowpark/modin/plugin/_internal/utils.py

```diff
@@ -248,7 +248,10 @@ def _create_read_only_table(
         ctas_query = f"SELECT * FROM {table_name}"
         temp_table_name = random_name_for_temp_object(TempObjectType.TABLE)
 
-        _logger.debug(f"Materialize temporary table {temp_table_name} for {ctas_query}")
+        _logger.warning(
+            f"Data from source table/view '{table_name}' is being copying into a new "
```

Comment:
can you also update one of our from_snowflake test to capture that this warning is logged